docs/en/guides/triton-inference-server.md

```diff
@@ -48,6 +48,16 @@ from ultralytics import YOLO
 # Load a model
 model = YOLO("yolo11n.pt")  # load an official model
 
+# Retreive metadata during export
+metadata = []
+
+
+def export_cb(exporter):
+    metadata.append(exporter.metadata)
+
+
+model.add_callback("on_export_end", export_cb)
+
 # Export the model
 onnx_file = model.export(format="onnx", dynamic=True)
```

Comment:
@Y-T-G can we directly use `metadata=model.exporter.metadate` here?

Reply:
@Laughing-q The exporter object is not stored, so we can't access it through the model. It's just directly created and called.

https://github.com/ultralytics/ultralytics/blob/21162bd870444550286983a601afbfb142f4c198/ultralytics/engine/model.py#L737

Reply:
@Y-T-G oh yeah you are right